File: models/estudiante.py
from sqlalchemy import Column, Integer, String, Boolean
from config.db import Base

class Estudiante(Base):
    __tablename__ = "estudiante"

    id_estudiante = Column(Integer, primary_key=True, index=True)
    nombres = Column(String(100), nullable=False)
    apellidos = Column(String(100), nullable=False)
    cedula = Column(String(20), unique=True, nullable=False)
    correo = Column(String(120), unique=True, nullable=False)
    contraseña = Column(String(255), nullable=False)
    telefono = Column(String(20))
    estado = Column(Boolean, default=True)

File: src/models/Estudiante.js
export class Estudiante {
  constructor({
    id_estudiante,
    nombres,
    apellidos,
    cedula,
    correo,
    telefono,
    estado,
  }) {
    this.id_estudiante = id_estudiante;
    this.nombres = nombres;
    this.apellidos = apellidos;
    this.cedula = cedula;
    this.correo = correo;
    this.telefono = telefono;
    this.estado = estado;
  }
}
